src/tachimawari/joint/model/joint.cpp

```diff
@@ -21,17 +21,38 @@
 #include <string>
 #include <vector>
 
+#include "keisan/angle/angle.hpp"
 #include "tachimawari/joint/model/joint.hpp"
 
 namespace tachimawari::joint
 {
 
+int Joint::angle_to_value(double angle)
+{
+  return angle * TO_VALUE_RATIO;
+}
+
+double Joint::value_to_angle(int value)
+{
+  return value * TO_ANGLE_RATIO;
+}
+
 Joint::Joint(uint8_t joint_id, float position)
-: id(joint_id), position(position), p_gain(30.0), i_gain(30.0), d_gain(30.0)
+: id(joint_id), position(keisan::make_degree(position)), p_gain(30.0), i_gain(30.0), d_gain(30.0)
+{
+}
+
+Joint::Joint(uint8_t joint_id, keisan::Angle<float> position)
+: Joint(joint_id, position.degree())
 {
 }
 
 void Joint::set_position(float position)
+{
+  this->position = keisan::make_degree(position);
+}
+
+void Joint::set_position(keisan::Angle<float> position)
 {
   this->position = position;
 }
```

Comment:
let's add deprecation mark in the `set_position(float)`, and return `keisan::Angle` in the `get_position()`.